docs/examples/jetson/concurrency_and_dynamic_batching/trtis_model_repo_sample_1/peoplenet/labels.txt

```diff
@@ -0,0 +1,3 @@
+person
```

Comment:
Since the labels are identical please reuse the same file instead of N different files.